### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by guybedford**
This includes a number of readme updates - updating the feature list of the builds to the latest state, while also cutting back on the content and including the key features as the first thing in the readme.

/cc @joeldenning for further review before release here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,68 +5,34 @@
...
-> [ES module Shims](https://github.com/guybedford/es-module-shims), like SystemJS, provides workflows for import maps and other modules features, but on top of base-level modules support in browsers. The performance difference is because source rewriting happens in browser instead of ahead-of-time like SystemJS handles via the System module format.
-
 ## Overview
 
-[Introduction video](https://www.youtube.com/watch?v=AmdKF2UhFzw)
-
-SystemJS provides two hookable base builds:
-
-#### 1. s.js minimal loader
+SystemJS is a hookable standards-based module loader. It provides a workflow where code written for production workflows of native ES modules in browsers ([like Rollup code-splitting builds](https://rollupjs.org/guide/en#code-splitting)), can be transpiled to the [System.register module format](docs/system-register.md) to work in older browsers that don't support native modules, running [almost-native module speeds](#performance) while supporting top-level await, dynamic import, circular references and live bindings, import.meta.url, module types, import maps, integrity and Content Security Policy with compatibility in older browsers back to IE11.
```
Comment (joeldenning) [inline README.md:10]:
```suggestion
SystemJS is a hookable, standards-based module loader. It provides a workflow where code written for production workflows of native ES modules in browsers ([like Rollup code-splitting builds](https://rollupjs.org/guide/en#code-splitting)), can be transpiled to the [System.register module format](docs/system-register.md) to work in older browsers that don't support native modules, running [almost-native module speeds](#performance) while supporting top-level await, dynamic import, circular references and live bindings, import.meta.url, module types, import maps, integrity and Content Security Policy with compatibility in older browsers back to IE11.
```

File: README.md [hunk 2]
```diff
@@ -5,68 +5,34 @@
...
-* Loads System.register modules.
-* Core hookable extensible loader supporting [custom extensions](docs/hooks.md).
+* Loads `System.register` modules, the CSP-compatible [SystemJS module format](docs/system-register.md).
+* Support for loading bare specifier names with [import maps](docs/import-maps.md) via `<script type="systemjs-importmap">`.
+* Supports [hooks](docs/hooks.md) for loader customization.
 
 #### 2. system.js loader
 
-The [3.7KB system.js loader](dist/system.min.js) loader builds on the s.js core and adds support for [global loading](#extras), [non-javascript module](/docs/module-types.md), and the [SystemJS registry API](/docs/api.md#registry).
+The [4.2KB system.js loader](dist/system.min.js) adds the following features in addition to the `s.js` features above:
```
Comment (joeldenning) [inline README.md:22]: Are we planning on removing the `dist` directory from source control in the 7.0 release?

File: README.md [hunk 3]
```diff
@@ -86,6 +52,28 @@ The following extras are included in system.js loader by default, and can be add
 
 Since all loader features are hookable, custom extensions can be easily made following the same approach as the bundled extras. See the [hooks documentation](docs/hooks.md) for more information.
 
+## SystemJS Babel
```
Comment (joeldenning) [inline README.md:55]: I'm excited to try this out!

**Review by joeldenning — APPROVED** [review]
Looks good - just minor comments.

File: README.md (hunk 2 above)
Comment (guybedford) [reply README.md:22]: Yeah that would be nice to do, when we get to that release.

**Pull request merged**